src/media/js/elements/nav.js

```diff
@@ -151,6 +151,13 @@ define('elements/nav',
                         if (this.isLeafPage) {
                             logger.log('hambacker button pressed (back)');
                             navigation.back();
+                        } else if (this.isCategoryPage) {
```

Comment:
this logic does not seem like it should be a part of the main nav component since it's baking in fireplace stuff, but practically it's not a big issue.